src/app/modules/core/core.errors.ts

```diff
@@ -63,3 +63,12 @@ export class MissingFeatureError extends ApplicationError {
     )
   }
 }
+
+/**
+ * Error thrown in testing code.
+ */
+export class TestError extends ApplicationError {
```

Comment:
since this is only relevant to tests, and specifically to the Analytics module, would it be better to move this to the folder containing the analytics tests?